assets/components/pages/project/curate/VariantData.js

```diff
@@ -119,6 +122,149 @@ TagsList.propTypes = {
   ).isRequired,
 };
 
+const BarChart = ({ data, options }) => {
+  const chartRef = useRef(null);
+  const chartInstance = useRef(null);
+
+  useEffect(() => {
+    if (chartRef && chartRef.current) {
+      chartInstance.current = new Chart(chartRef.current, {
```

Comment:
Oh, I think I misunderstood, the reactjs-charts2 was a React wrapper of the charts.js library, with this information I think your original approach was actually better.

Sorry, I didn't realise that charts.js was going to have anti-react patterning.